[2021-09-01 03:18:45,178] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: xcom_dag.processing_tasks.training_model_a 2021-09-01T03:18:37.836550+00:00 [queued]>
[2021-09-01 03:18:45,304] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: xcom_dag.processing_tasks.training_model_a 2021-09-01T03:18:37.836550+00:00 [queued]>
[2021-09-01 03:18:45,305] {taskinstance.py:1067} INFO - 
--------------------------------------------------------------------------------
[2021-09-01 03:18:45,307] {taskinstance.py:1068} INFO - Starting attempt 1 of 1
[2021-09-01 03:18:45,310] {taskinstance.py:1069} INFO - 
--------------------------------------------------------------------------------
[2021-09-01 03:18:45,421] {taskinstance.py:1087} INFO - Executing <Task(PythonOperator): processing_tasks.training_model_a> on 2021-09-01T03:18:37.836550+00:00
[2021-09-01 03:18:45,459] {standard_task_runner.py:52} INFO - Started process 116067 to run task
[2021-09-01 03:18:45,478] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'xcom_dag', 'processing_tasks.training_model_a', '2021-09-01T03:18:37.836550+00:00', '--job-id', '36', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/xcom_dag.py', '--cfg-path', '/tmp/tmp1swnz27v', '--error-file', '/tmp/tmppq085r3k']
[2021-09-01 03:18:45,485] {standard_task_runner.py:77} INFO - Job 36: Subtask processing_tasks.training_model_a
[2021-09-01 03:18:45,935] {logging_mixin.py:104} INFO - Running <TaskInstance: xcom_dag.processing_tasks.training_model_a 2021-09-01T03:18:37.836550+00:00 [running]> on host airflowvm
[2021-09-01 03:18:46,552] {taskinstance.py:1280} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=xcom_dag
AIRFLOW_CTX_TASK_ID=processing_tasks.training_model_a
AIRFLOW_CTX_EXECUTION_DATE=2021-09-01T03:18:37.836550+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-09-01T03:18:37.836550+00:00
[2021-09-01 03:18:46,571] {logging_mixin.py:104} INFO - model's accuracy: 1.0390884416553492
[2021-09-01 03:18:46,763] {python.py:151} INFO - Done. Returned value was: None
[2021-09-01 03:18:46,847] {taskinstance.py:1184} INFO - Marking task as SUCCESS. dag_id=xcom_dag, task_id=processing_tasks.training_model_a, execution_date=20210901T031837, start_date=20210901T031845, end_date=20210901T031846
[2021-09-01 03:18:47,111] {taskinstance.py:1245} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2021-09-01 03:18:47,177] {local_task_job.py:151} INFO - Task exited with return code 0
